docs_outdoornav_user_manual/api/api_endpoints.mdx

```diff
@@ -842,14 +1067,112 @@ string firmware_revision
 
 #### cpr_safety_msgs/WatchdogStatus {#msg-safety-watchdog}
 
-``` 
+```
 # Watchdog status message containing information related to sensor monitoring, emergency stop and safety stops
 
 bool[] gps_watchdog_triggered
 bool[] lidar_watchdog_triggered
 bool[] camera_watchdog_triggered
 ```
 
+#### onav_mission_manager_msgs/msg/Mission {#storage-mission}
+
+```
+# A mission containing one or more Waypoints, which may contain zero or more Tasks
+
+# A UUID string used to uniquely identify this mission
+string uuid
+
+# The human-readable name of this mission
+string name
+
+# The ordered list of Waypoints that make up the mission
+onav_mission_manager_msgs/Waypoint[] waypoints
+
+# Configuration parameters for the mission
+# Additional details TBD

```

Comment:
"TBD": is there more work to be done here?

Reply:
I don't have any details about the contents of the `onav_config` string; I'm not sure what its expected contents are or if it's even currently used or if it's just there for future expansion.  @jmastrangelo-cpr can probably elaborate here.

Reply:
Confirmed with Steve; it's not currently used.  Implementation is planned for 0.8 or 0.9. Updated the comment accordingly.